api/GeolocationCoordinates.json

```diff
@@ -146,7 +156,7 @@
               "version_added": "5"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "5"
```

Comment:
The above edit says 4.2, and api.Navigator.geolocation does too. But https://caniuse.com/geolocation suggests this was shipped on iOS before macOS, which actually makes a lot of sense for this API. Perhaps leave the iOS data alone, and do a separate PR to make it consistent?

Reply:
For the avoidance of doubt: I'm waiting for this before reviewing again.

Reply:
It does seem like support on iOS has been around a lot longer per testing in BrowserStack on iOS 3.  I'll set this to `≤3` for iOS accordingly, and update the other Geolocation APIs as well.

Reply:
Wohoo!